www/main/addons/ceremonies/components/CeremonyPane.jsx

```diff
@@ -25,10 +27,22 @@ const CeremonyPane = props => {
     applyTheme(theme);
   };
 
-  const toggleEnglishExplainations = isEnglishExplainations => {
-    store.setUserPref(`gurbani.ceremonies.ceremony-${token}-english`, isEnglishExplainations);
+  const toggleOptions = (toggleType, toggleVar) => {
+    store.setUserPref(`gurbani.ceremonies.ceremony-${token}-${toggleType}`, toggleVar);
     global.platform.updateSettings();
-    loadCeremonyGlobal(id);
+    const ceremonyToLoad =
+      toggleType === 'rm' && !toggleVar ? ceremoniesFilter.raagmalaMap[id] : id;
```

Comment:
when toggling, the history is updated. probably shouldn't but the entry in the history for raagmala off is an empty string. 
![image](https://user-images.githubusercontent.com/2982784/97372196-494b2f80-1870-11eb-9c32-2ef2f66ce0f7.png)

Reply:
@tsingh777 figured out the problem, the database does not have a Gurmukhi toggle for this ceremony. Preventing addition in history could be the way to go - but that would be pretty hacky as for all intents and purposes on technical side, its a separate ceremony.

Reply:
As discussed, we'll fix the DB entry.

Reply:
Also it should work now with rm toggle off https://youtu.be/Xd9GrcOJX8Q